src/main/groovy/au/org/ala/spatial/process/AooEoo.groovy

```diff
@@ -54,6 +54,11 @@ class AooEoo extends SlaveProcess {
 
         // eoo
         def points = facet("lat_long", speciesArea)
+
+        if(points.size() == 0){
```

Comment:
[ALA coding style](https://docs.google.com/document/d/1AMGskNqG5yxShiGsfnUj7N1zYvQCDyp5TTOyFakNO8Y/edit#heading=h.8zmpjqdoymld) is to include space between `if` and `(`.